packages/core/src/internal/utils/global.ts

```diff
@@ -50,26 +41,41 @@ export function setupCDSGlobal() {
   if (isBrowser()) {
     initializeCDSGlobal();
     setRunningVersion();
+    intializeCDSStateProxy();
   }
 }
 
-function getVersion() {
-  if (isBrowser()) {
-    const log: CDSLog = {
-      versions: window.CDS._version,
-      environment: window.CDS.environment,
-      userAgent: navigator.userAgent,
-      supports: window.CDS._supports,
-      angularVersion: getAngularVersion(false),
-      angularJSVersion: getAngularJSVersion(false),
-      reactVersion: getReactVersion(false),
-      vueVersion: getVueVersion(false),
-      loadedElements: window.CDS._loadedElements,
-    };
-    return log;
+function intializeCDSStateProxy() {
+  if (!window.CDS._stateProxy) {
+    window.CDS._stateProxy = true;
```

Comment:
It seems like `_stateProxy` isn't a proxy but instead notes whether the `_state` object has been proxied?

I was expecting this property to be the proxy at first. Maybe we can update that to be `_stateProxied` or `_isStateProxied`?

Reply:
updated